packages/ses/package.json

```diff
@@ -5,11 +5,10 @@
   "author": "Agoric",
   "license": "Apache-2.0",
   "type": "module",
-  "main": "./src/lockdown-shim.js",
+  "main": "./dist/ses.cjs",
   "exports": {
-    "import": "./src/lockdown-shim.js",
-    "require": "./dist/ses.cjs.js",
-    "browser": "./dist/ses.umd.js"
```

Comment:
Can you please preserve the `"browser"` field?  It supports `<script>` tags.

Reply:
What systems respect the browser field? Is this an aspirational predicate; something future systems might use? Ideally this would be a testable case, though I’m sure that might be too much to ask.

Reply:
Aspirational, I think.